Deployment/Jenkins/parent-autoscale.groovy

```diff
@@ -29,65 +29,65 @@ pipeline {
-        string(name: 'ElbTemplate', description: 'URI for the template that creates the ELB providing public-to-private reverse-proxy service')
-        string(name: 'IamTemplate', description: 'URI for the template that creates SonarQubes IAM role(s)')
-        string(name: 'RdsTemplateUri', description: 'URI for the template that creates SonarQubes RDS database')
-        string(name: 'SgTemplateUri', description: 'URI for the template that creates SonarQubes security groups')
-        string(name: 'SonarOsPrepUrl', description: 'URL of the script ready the OS for Sonarqube installation')
-        string(name: 'SonarAppSetupUrl', description: 'URL of the Sonarqube setup script')
-        string(name: 'PluginsBucket', description: 'S3 bucket hosting Sonarqube installations feature-extending plugins')
-        string(name: 'PluginFolder', description: 'S3-path containing Sonarqube plugins')
         string(name: 'AdminPubkeyURL', description: 'URL the file containing the admin users SSH public keys')
-        string(name: 'PipRpm', defaultValue: 'python2-pip', description: 'Name of preferred pip RPM')
-        string(name: 'ServiceTld', defaultValue: 'amazonaws.com', description: 'TLD of the created EFS-endpoint')
-        string(name: 'SonarqubeListenerCert', defaultValue: '', description: 'Name/ID of the ACM-managed SSL Certificate to protect public listener')
+        string(name: 'AmiId', description: 'ID of the AMI to launch')
         string(name: 'BackendTimeout', defaultValue: '600', description: 'How long - in seconds - back-end connection may be idle before attempting session-cleanup')
-        string(name: 'DbInstanceName', description: 'Instance-name of the SonarQube database')
-        string(name: 'DbNodeName', description: 'NodeName of the SonarQube database')
+        string(name: 'BucketInventoryTracking', defaultValue: 'false', description: '(Optional) Whether to enable generic bucket inventory-tracking. Requires setting of the ReportingBucket parameter')
+        string(name: 'BucketTemplate', description: 'URI for the template that creates SonarQubes S3 bucket(s)')
         string(name: 'DbAdminName', description: 'Name of the SonarQube master database-user')
         string(name: 'DbAdminPass', description: 'Password of the SonarQube master database-user')
-        string(name: 'DbInstanceType', defaultValue: 'db.t2.small', description: 'Amazon RDS instance type')
         string(name: 'DbDataSize', defaultValue: '5', description: 'Size in GiB of the RDS table-space to create')
+        string(name: 'DbInstanceName', description: 'Instance-name of the SonarQube database')
+        string(name: 'DbInstanceType', defaultValue: 'db.t2.small', description: 'Amazon RDS instance type')
+        string(name: 'DbNodeName', description: 'NodeName of the SonarQube database')
         string(name: 'DbSnapshotId', defaultValue: '', description: '(Optional) RDS snapshot-ARN to clone new database from')
```

Comment:
Might need to figure out how to mention that when this value is non-null, the `DbAdminName` and `DbAdminPass` must match the value stored in the snapshot (failure to match is quite "no bueno")

Reply:
Ok, I added some BS. Not sure it's final BS but at least we're tracking